lib/facility.rb

```diff
@@ -1,14 +1,51 @@
+<<<<<<< HEAD
```

Comment:
Same goes for this class --> you're still in the midst of a merge conflict here and will need to merge the code accordingly and then remove the merge conflict symbols